diagnostic_feedback/templates/student/quiz.html

```diff
@@ -0,0 +1,53 @@
+<div class="diagnostic-feedback diagnostic-feedback-student">
+    <div class="msg"><h3></h3></div>
+    <form id="student_view_form">
+        {% if user_is_staff %} <a href="#" id="export_data">Download Data</a><span id="export_progress"></span> {% endif %}
+        {% if questions %}
+            <h1 class="quiz-title"><i class="icon fa fa-list"></i> {{self.title}} </h1>
+            <h3 class="quiz-description">{{self.description|safe}}</h3>
+            <div class="question-div">
+                <input type="hidden" name="completed_step" id="completed_step" value="{{self.current_step}}">
+                {% for question in questions %}
+                <h3 style="display:none;">Question</h3>
```

Comment:
h3 is required for jQuery.steps plugin as it's using it to generate heading HTML. We can remove inline style and it's removed now.